snprc_ehr/src/org/labkey/snprc_ehr/controllers/FeeScheduleController.java

```diff
@@ -146,24 +153,53 @@ public ModelAndView getView(FeeScheduleImportForm form, boolean reshow, BindExce
             return new JspView<>("/org/labkey/snprc_ehr/pipeline/FeeScheduleImport.jsp", form, errors);
         }
 
+        private File getVerifiedFile(String filePath)
```

Comment:
@labkey-matthewb Would it be generally useful to have this a helper method on the Piperoot/PipelineService? Adding either the root or the container as a parameter makes this validation pretty generic for parsing incoming 
file paths.

Reply:
@labkey-ians  I was wondering the same